contracts/Optimism_SpokePool.sol

```diff
@@ -15,11 +19,17 @@ import "./SpokePoolInterface.sol";
 
 contract Optimism_SpokePool is CrossDomainEnabled, SpokePoolInterface, SpokePool, Ownable {
     // "l1Gas" parameter used in call to bridge tokens from this contract back to L1 via `IL2ERC20Bridge`.
-    uint32 l1Gas = 6_000_000;
+    uint32 public l1Gas = 1_900_000;
```

Comment:
This is L1 gas. Are you sure this should be set to 1.9mil? I thought only the L2 gas param in the Optimism Adapter (which calls L1--> L2 txns) should use this value

Reply:
I'm not exactly sure. tbh I'm not exactly sure how it works going from L2->L1, only really know what the costing is from L1->L2.

Reply:
I think you can leave this at 6mil, but +1 on making it `public`